apps/bmd/src/SampleApp.tsx

```diff
@@ -24,12 +26,15 @@ export function getSampleCard(): Card {
 }
 
 export function getSampleStorage(): Storage<AppStorage> {
-  const election = electionSample
+  const electionDefinition = {
+    election: electionSample,
+    electionHash: sha256(JSON.stringify(electionSample)),
```

Comment:
This one doesn't matter as much since it's just the sample app, but we should probably change ballot-encoder to export an `ElectionDefinition` to ensure the hash is correct.